js/chatbot/static/Copy.svelte

```diff
@@ -46,30 +45,25 @@
 </script>
 
 <button on:click={handle_copy} title="copy">
-	<span class="copy-text" class:copied><Copy /> </span>
```

Comment:
this logic seemed to be unmaintained - it was not working. I simplified and removed all the outdated and non-necessary parts.